<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetDataWidget</class>
 <widget class="QDialog" name="SetDataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="title_lbl">
       <property name="text">
        <string>DH model paramemters</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableData_tw">
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>a(m)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>d(m)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>theta (rad)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>alpha (rad)</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="result_lbl">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addRow_pb">
       <property name="text">
        <string>Add Row</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeRow_pb">
       <property name="text">
        <string>Remove Row</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="startCalc_pb">
     <property name="text">
      <string>Start Calculation</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
